### PR TITLE
fix test suite by declaring a deleter for CDataCacheCore

### PR DESCRIPTION
**Pull request by FernetMenta**
see title

### CODE REVIEW

File: xbmc/ServiceManager.cpp [hunk 1]
```diff
@@ -110,4 +110,10 @@ ActiveAE::CActiveAEDSP& CServiceManager::GetADSPManager()
 CDataCacheCore& CServiceManager::GetDataCacheCore()
 {
   return *m_dataCacheCore;
-}
\ No newline at end of file
+}
+
+// delters for unique_ptr
```
Comment (hudokkow) [inline xbmc/ServiceManager.cpp]: s/delters/deleters

**Comment by FernetMenta** [comment]
jenkins build this please

**Comment by Paxxi** [comment]
I have a feeling that this hides the real issue as we really shouldn't have to specify a deleter. Poking around with it I can't find anything that seems wrong that would cause the original issue though so we should be fine.

**Comment by FernetMenta** [comment]
As @notspiff said, either include the header in ServiceManager.h or specifying a deleter.

**Comment by Paxxi** [comment]
Ahh I missed that comment, anyway issue solved, thanks

**Pull request merged**